test_quick_sort.py

```diff
@@ -0,0 +1,40 @@
+from random import shuffle
+import pytest
+
+from quick_sort import quick_srt
+
+
+def test_quick_srt():
+    expected = range(20)
+    actual = expected[:]
+    shuffle(actual)
```

Comment:
This is probably fine, but make sure you also have reproducible tests. Maybe a 2nd test that always uses the same un-sorted list.